tests/fixtures/expected/main/datetime.html

```diff
@@ -190,10 +190,13 @@ <h3 id="error-bubbling"><a class="headerlink" href="#error-bubbling" title="Perm
 </div>
 <p>And configuration blocks:</p>
 <div class="configuration-block">
-    <ul class="simple">
-                    <li>
-                <em>YAML</em>
-                <div translate="no" class="notranslate codeblock codeblock-loc-1 codeblock-yaml">
+    <ul class="configuration-tabs">
+        <li data-language="YAML" data-active="true"> <span>YAML</span> </li>
+        <li data-language="XML" > <span>XML</span> </li>
+        <li data-language="PHP" > <span>PHP</span> </li>
```

Comment:
```diff
        <li data-language="PHP" ><span>PHP</span></li>
```
Do we really need to keep these spaces?

Reply:
I'd prefer to keep it (it's the same case as https://github.com/weaverryan/docs-builder/pull/56#issuecomment-791463260).

I know some people disagree but ... I prefer to make templates readable instead of making the generated HTML perfect, because templates are for humans (and they must maintain them in the future) and HTML is for machines and they don't care.

I hope you agree somewhat on this 🙏

Reply:
:+1: to be pragmatic on this kind of things